shell.nix

```
@@ -0,0 +1,23 @@
+with import <nixpkgs> { };
+
+let
+  moz_overlay = import (builtins.fetchTarball
+    "https://github.com/mozilla/nixpkgs-mozilla/archive/master.tar.gz");
+  nixpkgs = import <nixpkgs> { overlays = [ moz_overlay ]; };
+in mkShell rec {
+  name = "eww";
+
+  buildInputs = [
+    glib
+    cairo
+    atk
+    gdk-pixbuf
+    pango
+    gtk3
+    (nixpkgs.rustChannelOf {
+      date = "2021-03-16";
+      channel = "nightly";
+    }).rust
+  ];
+  nativeBuildInputs = [ pkgconfig wrapGAppsHook ];
```

```suggestion
    ];
	  nativeBuildInputs = [
	    (nixpkgs.rustChannelOf {
	      date = "2021-03-16";
	      channel = "nightly";
	    }).rust
	    pkgconfig
	    wrapGAppsHook
	  ];
```

build time dependencies should be in `nativeBuildInputs`

---

It won't affect anything now but when the wayland support get merged `gtk-layer-shell` will need to be added to the buildInputs.

---

Wayland has now been merged, so this is now a thing that should be considered